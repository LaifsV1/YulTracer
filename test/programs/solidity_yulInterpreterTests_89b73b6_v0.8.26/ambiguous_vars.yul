{
  {
     let a := 0x20
     mstore(a, 2)
  }
  let a
  mstore(a, 3)

  // Added memory dump:
  printhex(mload(0x0))
  printhex(mload(0x20))
}
// ----
// Trace:
// Memory dump:
//      0: 0000000000000000000000000000000000000000000000000000000000000003
//     20: 0000000000000000000000000000000000000000000000000000000000000002
// Storage dump:
// Transient storage dump:
