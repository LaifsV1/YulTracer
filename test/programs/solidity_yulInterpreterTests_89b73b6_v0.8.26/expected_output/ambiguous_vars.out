0000000000000000000000000000000000000000000000000000000000000003
0000000000000000000000000000000000000000000000000000000000000002
